### PR TITLE
Support manage curve monitor, curve website and support http service

### PR DESCRIPTION
**Pull request by SeanHai**

### CODE REVIEW

File: internal/configure/monitor.go [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+	KEY_HOST              = "host"
+	KEY_LISTEN_PORT       = "listen_port"
+	KEY_RETENTION_TIME    = "retention.time"
+	KEY_RETENTION_SIZE    = "retention.size"
+	KEY_PROMETHEUS_TARGET = "target"
+	KEY_GRAFANA_USER      = "username"
+	KEY_GRAFANA_PASSWORD  = "password"
+
+	KEY_NODE_IPS         = "node_ips"
+	KRY_NODE_LISTEN_PORT = "node_listen_port"
```
Comment (caoxianfei1) [inline internal/configure/monitor.go:54]: KEY_NODE_LISTEN_PORT

File: cli/command/monitor/deploy.go [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+	for _, dc := range dcs {
+		thostMap[dc.GetHost()] = true
+		thostIpMap[dc.GetListenIp()] = true
+	}
+	for key := range thostMap {
+		hosts = append(hosts, key)
+	}
+	for key := range thostIpMap {
+		hostIps = append(hostIps, key)
+	}
```
Comment (caoxianfei1) [inline cli/command/monitor/deploy.go:129]: thostMap and thostIpMap are redundant?

File: internal/task/task/monitor/create_container.go [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+		Image:       cfg.GetImage(),
+		Command:     getArguments(cfg),
+		AddHost:     []string{fmt.Sprintf("%s:127.0.0.1", hostname)},
+		Envs:        getEnvironments(cfg),
+		Hostname:    hostname,
+		Init:        true,
+		Name:        hostname,
+		Privileged:  true,
+		User:        "0:0",
+		Pid:         "host",
```
Comment (caoxianfei1) [inline internal/task/task/monitor/create_container.go:153]: Pid using `host` will share host progress.

File: internal/configure/monitor.go (hunk 1 above)
Comment (SeanHai) [reply internal/configure/monitor.go:54]: fixed

File: cli/command/monitor/deploy.go (hunk 2 above)
Comment (SeanHai) [reply cli/command/monitor/deploy.go:129]: They are used to deduplicat hostname and ip

File: internal/task/task/monitor/create_container.go (hunk 3 above)
Comment (SeanHai) [reply internal/task/task/monitor/create_container.go:153]: Yes, now the container created by curveadm are all 'host' type.

**Review by caoxianfei1 — APPROVED** [review]

**Pull request merged**